src/github/index.js

```diff
@@ -12,8 +12,8 @@ const ISSUE_STATE_ALL = 'all';
 
 const UPDATE_DOCUMENT_LABEL = process.env.GITHUB_LABEL_UPDATE || 'update';
 
-const LOCAL_CONTRIBUTE_URL = 'http://localhost:3000/contribute/service';
-const CONTRIBUTE_URL = 'https://opentermsarchive.org/en/contribute/service';
+const LOCAL_CONTRIBUTE_URL = `http://localhost:3000/contribute/service?${process.env.GITHUB_REPO}`;
```

Comment:
Set where your construct the `urlQueryParams` string instead of here. Moreover an `&` is missing:

```
const urlQueryParams = `${process.env.GITHUB_REPO}&step=2&url=${encodedUrl}&name=${encodedName}&documentType=${encodedType}${noiseSelectorsQueryString}${contentSelectorsQueryString}&expertMode=true`;
```

Reply:
Also I see you did not give a name to your query param. I think `repository=${process.env.GITHUB_REPO}` should be better

Reply:
You're all right! (late commit after a too long day)

Everything is fixed